### DISCUSSION TITLE
Roomcard Temperature + Humidity

### DISCUSSION THREAD

**Discussion by Does-This-Work** [UI-Lovelace-Minimalist/UI#1183] [Q&A] [answered]
I am looking for a way to include a temperature and humidity entity in the roomcard. Currently I can only display either temperature or humidity. 
![grafik](https://user-images.githubusercontent.com/102190830/221210916-28523327-a8ea-40dc-913d-529d7b99516f.png)
```
  - type: 'custom:button-card'
	template:
	  - card_room
	  - red_no_state
	name: Bedroom
	entity: sensor.temperature
	icon: mdi:bed-king
	tap_action:
		action: navigate
		navigation_path: '/ui-lovelace-minimalist/bedroom'
	variables:
	  label_use_temperature: true
	  label_use_brightness: false
	  entity_1:
		entity_id: light.ceiling
		templates:
		  - yellow_on
		tap_action:
		  action: toggle
	  entity_2:
		  entity_id: cover.curtain
		  templates:
			- yellow_on
		  tap_action:
			action: toggle 
```
My idea was to combine to entities, but that didn't work like I imagend. I would like to add some other values too, since there are more values to a room than just temperature.
```
  - type: 'custom:button-card'
	template:
	  - card_room
	  - red_no_state
	name: Bedroom
	entity: {{ [sensor.temperature], [sensor.humidity] }}
	icon: mdi:bed-king
```
I tried the things they suggest in this discussion but I believe it only works with a string. [https://github.com/UI-Lovelace-Minimalist/UI/discussions/1148](url)

**Comment by Does-This-Work** [comment, marked as answer]
#1181 thanks to the help of @antonio1475 I created my own custom room card.
The first entity is the humidity sensor and the second one is the temperature sensor

![grafik](https://user-images.githubusercontent.com/102190830/221935618-56309fc4-d8aa-4c75-a39a-120fe85f0abf.png)


```
  variables:
    label_use_brightness: false
    label_use_temperature: false
    label_use_temperature_entity:
```
```
  label: |-
    [[[
      var label = ""
      
      if (variables.label_use_temperature ) {
        label += (states[variables.label_use_temperature_entity].attributes.current_temperature || states[variables.label_use_temperature_entity].attributes.temperature || states[variables.label_use_temperature_entity].state || entity?.attributes?.current_temperature || entity?.attributes?.temperature || entity?.state || '-') + ('°C');
        
      }
      
      label += " | " + variables.ulm_translation_state + "%";
      
      return label;
    ]]]
```

**Reply by stefano-ita** [reply]
Hello there,
I tried this code to have temperature and humidity together but it doesn't work...
Do you have any suggestions about why?
Thanks in advance,
Stefano

**Reply by stefano-ita** [reply]
Solved with this:
`label: '[[[ return + states["sensor.sonoff_a48005998a_temperature"].state +"°C  | "+states["sensor.sonoff_a48005998a_humidity"].state + "%"]]]'`
Thanx to the forum!

**Reply by raider206** [reply]
I have the same card with temperature and humidity since some months ago, but now I have the problem that they appear with a lot of decimals...
I have this: 
label: >
[[[
    return "🌡️ " + states['sensor.ble_sensor_living_temperature'].state + " °C" 
          + " 💧 " + states['sensor.ble_sensor_living_humidity'].state + " %"
]]]

And appears this:
![image](https://github.com/UI-Lovelace-Minimalist/UI/assets/73711745/ed088f6a-5932-4723-b0a5-47f69d38e9b9)

What can I do to round to 1 decimal ??

Thank you!
